python/ray/rllib/ddpg/common/wrappers.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ray.rllib.models import ModelCatalog
+from ray.rllib.utils.atari_wrappers import wrap_deepmind
+
+
+def wrap_ddpg(registry, env, options, random_starts):
+    """Apply a common set of wrappers for DDPG."""
+
+    is_atari = hasattr(env.unwrapped, "ale")
+
+    # Override atari default to use the deepmind wrappers.
+    # TODO(ekl) this logic should be pushed to the catalog.
+    if is_atari and "custom_preprocessor" not in options:
+        return wrap_deepmind(env, random_starts=random_starts)
```

Comment:
Since DDPG isn't typically used with discrete action spaces (e.g, atari), how about we remove this wrapper and just use `ModelCatalog.get_preprocessor...`?

This means we can remove this file.

Reply:
I tried your solution. However, without an complicated assertion on observation type, say Box(0.0, 1.0, (80, 80, 1), dtype=np.float32) is allowed while Box(0.0, 1.0, (210, 160, 3), dtype=np.float32) is unsupported, DDPG can NOT pass the test/test_supported_spaces.py test. Thus, I removed this file according to your comment but directly import and use it from dqn instead of your proposal. Any concern, please let me know and I will revise according to your comments.

Reply:
I see, that sounds fine then, we can clean up Atari handling later.